real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java

```diff
@@ -28,8 +27,7 @@ public static void main(String[] args) {
 		RobobuggyConfigFile.loadConfigFile(); //TODO make sure that logic Notification is setup before this point
 
 		new RobobuggyLogicNotification("Initializing Robot", RobobuggyMessageLevel.NOTE);
-        robot = SimRobot.getInstance();
-
+		robot = TransistorAuton.getInstance();
```

Comment:
Can we change it s.t. we're not selecting the robot in the code?  I recall having to constantly check the code before we went on test runs